addons/website/static/src/components/website_loader/website_loader.js

```diff
@@ -101,10 +106,15 @@ export class WebsiteLoader extends Component {
         useBus(this.props.bus, "SHOW-WEBSITE-LOADER", (ev) => {
             const props = ev.detail;
             this.state.isVisible = true;
-            this.state.title = props && props.title;
-            this.state.showTips = props && props.showTips;
-            this.state.selectedFeatures = props && props.selectedFeatures;
-            this.state.showWaitingMessages = props && props.showWaitingMessages;
+            if (props) {
+                this.state.title = props.title;
+                this.state.showTips = props.showTips;
+                this.state.selectedFeatures = props.selectedFeatures;
+                this.state.showWaitingMessages = props.showWaitingMessages;
+                this.state.bottomMessage = props.bottomMessage;
+                this.state.showLoader = props.showLoader !== false;
+                this.state.showCloseButton = props.showCloseButton;
+            }
```

Comment:
Should not we just remove the `if`? Can it be falsy? Because if it can, then this may be wrong: you would want to set up the state values as falsy too?

Reply:
Should be good now! :sparkles: